docs/api-graduation-guidelines.md

```diff
@@ -0,0 +1,31 @@
+This document defines the expectations of how API graduation is handled across the github.com/kubevirt organization. Individual projects within the KubeVirt org may decide to append additional criteria and timeline constraints to the items outlined here, but should not deviate from the broad expectations as defined in this document.
+
+# Graduation Phase Expecations
+
+## Alpha
+* Feature incomplete, meaning the API might be half implemented as functionality lands in phases.
+* Basic testing exists proving that the merged functionality works e2e.
```

Comment:
Basic testing exists proving that the merged functionality works **for e2e tests to ensure the application flow behaves as expected**.

Reply:
I'm not sure I manage to parse your suggested sentence @marceloamaral. Maybe
```
End-to-end tests are merged, proving that the application flow keeps behaving as expected
```

Reply:
i reworded this a bit